WordPress/src/test/java/org/wordpress/android/ui/mysite/MySiteViewModelTest.kt

```diff
@@ -956,6 +959,17 @@ class MySiteViewModelTest : BaseUnitTest() {
         verify(quickStartRepository).refresh()
     }
 
+    @Test
+    fun `given build is Jetpack, then quick action block is not built`() {
```

Comment:
1) Praise (❤️): Nice addition!
2) Suggestion (💡): Since this `Jetpack` related test was added, I suggest adding the equivalent `WordPress` test to verify that, on the other way around, quick actions block is built (`isNotNull`). Wdyt?
3) Minor (🔍) The `given/when/then` pattern above is missing the `when` sentence. Usually this test sentence, when using this pattern, would have been written, kind of, as: `given jetpack app, when site items are built, then quick action block is not available`

Reply:
Thanks for your changes @zwarm and for merging! 🌟 

> I thought about that, but there are a dozen other tests that test that scenario and Ididn't feel it was warranted. I don't mind adding it, seriously it's a copy/paste and change true to false. :)

You are very correct. It just that when you then add one test that test half the code branch (`if/else`), I always feel obligated to included a test for the other branch, just to verify correctness on the first branch, especially when the assertion on the first branch is testing for something to be null, which might be valid even without any precondition or setup. Does this makes sense at all? 😅